src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1685,5 +1685,34 @@ sap.ui.define([
 		return this.bSecureSearch;
 	};
 
+	/**
+	 * @typedef {object} mParameters
```

Comment:
This name should follow the same syntax as sap.fhir.model.r4.RequestParameter. Please have a look also on ?`FHIRListBinding._buildParameters` which generates an instance using this type. Should be aligned accordingly. Also see that `FHIRTreeBinding._buildParameters` also uses this kind of functionality but with less parameter.
I would recommend to name the type `sap.fhir.model.r4.FHIRListBinding.Parameter` and `sap.fhir.model.r4.FHIRTreeBinding.Parameter`

Reply:
done